Topics/Tech_Stacks/DigitalOcean_Web_App.md

```diff
@@ -0,0 +1,129 @@
+# Deploying a Web Application on DigitalOcean
+
+## Introduction
```

Comment:
I think this section sound a bit repetitive, as it talks about deploying a web application using digital ocean, a cloud service for hosting applications.

Maybe begin with introducing what Digital Ocean is with a few more sentences, and then mentioning how this tutorial will guide you through deploying a web application.